src/test/isolation2/sql/vacuum_progress_row.sql

```diff
@@ -143,8 +143,17 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
 -- Resume walsender to detect mirror down and suspend at the beginning
 -- of post-cleanup taken over by a new vacuum worker.
 2: SELECT gp_inject_fault('vacuum_worker_changed', 'suspend', dbid) FROM gp_segment_configuration WHERE content > -1 AND role = 'p';
+
+-- we shouldn't proceed until FTS probe has bumped the version
+2: SELECT gp_inject_fault('vacuum_rel_finished_one_relation', 'suspend', '', '', 'vacuum_progress_ao_row', 1, 1, 0, 1) FROM master();
+2: SELECT gp_inject_fault('ftsLoop_after_probe', 'skip', 1) FROM master();
 -- resume walsender and let it exit so that mirror stop can be detected
 2: SELECT gp_inject_fault_infinite('wal_sender_loop', 'reset', dbid) FROM gp_segment_configuration WHERE role = 'p' and content = 1;
+-- wait for FTS probe
+2: SELECT gp_wait_until_triggered_fault('ftsLoop_after_probe', 1, 1) FROM master();
+2: SELECT gp_inject_fault('vacuum_rel_finished_one_relation', 'reset', 1) FROM master();
+2: SELECT gp_inject_fault('ftsLoop_after_probe', 'reset', 1) FROM master();
+
```

Comment:
Can we use `gp_request_fts_probe_scan()` instead of `gp_inject_fault`? It seems to do exactly what we need. Starts the fts probe and waits for it to complete.